django_ses_plus/tasks.py

```diff
@@ -23,19 +24,28 @@ def send_email(subject, to_email, html_message, from_email=None, message=None, r
         subject=subject,
         message=message,
         html_message=html_message,
+        attachments=attachments,
         from_email=from_email,
         recipient_list=[to_email],
         fail_silently=False,
     )
 
     try:
-        SentEmail.objects.create(
+        sent_email = SentEmail.objects.create(
```

Comment:
Should we add a transaction block here?

Reply:
This is the last operation. What will be the advantage?

Reply:
We can create the `SentEmail` successfully, but can get an error while creating `SentEmailAttachment`. Our record will be consistent which is advantage. But, we will lose the email record completely, sometimes you may want to achieve email object even if the attachment is misleading, this is disadvantage.

Reply:
We will have an error log for that. I think it's better to have the "sent email" record even though there are missing attachments.

Reply:
Makes sense thank you.